### PR TITLE
Feature/ar 11

### PR DESCRIPTION
**Pull request by golovnyov**

### CODE REVIEW

File: Backend/AnimalRescue/AnimalRescue.Infrastructure/Extensions.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace AnimalRescue.Infrastructure
+{
+    public static class Extensions
```
Comment (AlexeyDerepa) [inline Backend/AnimalRescue/AnimalRescue.Infrastructure/Extensions.cs]: name of your class is not show the main aim for what object it was implemented

File: Backend/AnimalRescue/AnimalRescue.DataAccess.Mongodb/Repositories/ConfigurationRepository.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    public class ConfigurationRepository : Repository<Configuration<CmsConfigurationNested>>, IConfigurationRepository
+    {
+        public ConfigurationRepository(IMongoClient client, IMongoDbSettings settings)
+            : base(client, settings)
+        {
+        }
+
+        public async Task<CmsConfigurationModel> GetCmsConfigurationAsync()
+        {
+            var configurationCursor = await collection.FindAsync(x => x.Name == "Cms");
```
Comment (AlistratovAlex) [inline Backend/AnimalRescue/AnimalRescue.DataAccess.Mongodb/Repositories/ConfigurationRepository.cs]: Move "Cms" to const

File: Backend/AnimalRescue/AnimalRescue.Contracts/IConfigurationService.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿using AnimalRescue.Models.DTO;
+using System.Threading.Tasks;
+
+namespace AnimalRescue.Contracts
+{
+    public interface IConfigurationService
+    {
+        Task<CmsConfigurationModel> GetCmsConfiguration();
```
Comment (AlistratovAlex) [inline Backend/AnimalRescue/AnimalRescue.Contracts/IConfigurationService.cs]: GetCmsConfiguration->GetCmsConfigurationAsync

Comment (golovnyov) [reply Backend/AnimalRescue/AnimalRescue.Contracts/IConfigurationService.cs]: Done

File: Backend/AnimalRescue/AnimalRescue.DataAccess.Mongodb/Repositories/ConfigurationRepository.cs (hunk 2 above)
Comment (golovnyov) [reply Backend/AnimalRescue/AnimalRescue.DataAccess.Mongodb/Repositories/ConfigurationRepository.cs]: Done

File: Backend/AnimalRescue/AnimalRescue.Infrastructure/Extensions.cs (hunk 1 above)
Comment (golovnyov) [reply Backend/AnimalRescue/AnimalRescue.Infrastructure/Extensions.cs]: Removed

File: Backend/AnimalRescue/AnimalRescue.DomainServices/AnimalRescue.BusinessLogic.csproj [hunk 4]
```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
```
[the remark this answers is not in the document]
Comment (golovnyov) [reply Backend/AnimalRescue/AnimalRescue.DomainServices/AnimalRescue.BusinessLogic.csproj]: Renamed

**Pull request merged**